proj/hps_accel/src/tensorflow/lite/kernels/internal/reference/integer_ops/fully_connected.h

```diff
@@ -16,8 +16,10 @@ limitations under the License.
 #define TENSORFLOW_LITE_KERNELS_INTERNAL_REFERENCE_INTEGER_OPS_FULLY_CONNECTED_H_
 
 #include <cstdio>
+#include <algorithm>
```

Comment:
@tcal-x is this the line triggering the litex issue btw?

Reply:
TFLM uses `<algorithm>` extensively (https://github.com/google/CFU-Playground/search?q=algorithm), so it is not just this one use.   This is just a case where its addition needed to be propagated into one of our local versions.